src/api/auth/changetemporaryKey.ts

```
@@ -0,0 +1,34 @@
+import axios, { AxiosResponse } from 'axios';
+import { auth } from '@/config';
+import { useMutation } from '@tanstack/react-query';
+interface auth {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'auth'. Consider to delete it or rename it.

'auth' is defined here:


(lint/suspicious/noRedeclare)